### ISSUE TITLE
1.5.10 (new) Simulator directory issue with 1.5.10 Pre-release

### ISSUE THREAD

**Issue by Alex00101** [FrSkyRC/ETHOS-Feedback-Community#4211]
**Background:**
When upgrading to 1.5.10 Nightly, noticed that scripts directory has been moved to the user directory, e.g.
   **C:\Users\My User\.ethosX10EXPRESS**
   **C:\Users\My User\.ethosX20PRO**
This was explained in another issue raised by someone else and subsequently closed. So I have updated all my build scripts accordingly, no problems there.

**Issue:**
This morning I upgraded to 1.5.10 Pre-release and noticed:

1. Simulator now is looking for scripts in .ethos/X10EXPRESS/ rather than .ethosX10EXPRESS
       **005254 f_opendir(C:/Users/MyUser/.ethos/X10EXPRESS/scripts) = error 2 (No such file or directory)**
2. Each time run the Simulator I'm getting the startup sequence (select the language, etc) then model wizard. If I create the model it does not seem to be saved anywhere.
3. There is no .ethos directory created in the C:\Users\My User folder, only previous .ethosX10EXPRESS, .ethosX20PRO
 
Hope is makes sense. For now I rolled back to the Nightly

**Comment by strgaltdel** [comment]
strange,

yesterday i installed x20s sim (Rev 2.July 8:11) 
today the preRelease in the same directory, no problems
I can still run older sims because they've the "old" environment in parallel

X20 pro Sim, todays pre release, installed from scratch, even is running without any problem.
but as you've seen on rcg, you're not alone, other users raised issues too...

maybe it helps the team if you post which nightly release you've used ?

**Comment by Alex00101** [comment]
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/27343158/218971d1-d90a-42ab-a109-a067bb9a15ac)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/27343158/3a749760-e16e-4e10-ac11-6862d92f953c)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/27343158/422d6a70-a235-4786-8f9e-7e1666fa10e6)

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/27343158/2a0dd3b8-5df7-414a-b0da-167cd697d6be)

**Comment by Alex00101** [comment]
Now this ts the fix:
- Go to the User directory (**C:\Users\My User**)
- Create **.ethos** folder under it
- Create **X20PRO** folder
- Copy the content of **.ethosX20PRO**

Run the same SIM as above, everything works

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/27343158/94ad83e4-a78b-4514-be78-199b7154ed8e)

**Comment by Alex00101** [comment]
Update: Also needed to copy the audio folder and its content from the installation directory (C:\Program Files (x86)\FrSky\Ethos\X20PRO) to C:\Users\My User\.ethos\X20PRO

Model bitmaps however need to stay in the installation directory.

I do hope this all gets fixed in 1.5.10 final

**Comment by dwheeld** [comment]
Is this the new file structure or were the new sims faulty?

**Comment by bsongis-frsky** [comment]
Both! It should be good now.
